machine-learning/lambda-v2/artifacts/com.lambda.DLR/1.6.0/installer/installer.sh

```diff
@@ -0,0 +1,65 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/bin/bash
+. $(dirname "$0")/utils.sh
+
+set -eux
+
+kernel=$(uname -s)
+dlr_version="1.6.0"
+min_py_version="3.0.0"
+dlr_directory="neo-ai-dlr"
+min_kernel_version="4.9.9"
+min_pip_version="20.2.4"
+machine=$(uname -m)
+python_version=$(echo $(python3 --version) | cut -d' ' -f 2)
+pip3_version=$(echo $(pip3 --version) | cut -d' ' -f 2)
+jetson_device_chip="/sys/module/tegra_fuse/parameters/tegra_chip_id"
+# Supported dlr pip wheels for different devices and dlr versions can be fount at
+# https://github.com/neo-ai/neo-ai-dlr/releases
+dlr_link="https://neo-ai-dlr-release.s3-us-west-2.amazonaws.com/v"$dlr_version
```

Comment:
I thought we were getting rid of these installers

Reply:
Yes, but we still need runtime to run these samples as is.  This is just a sample script to install runtime as well.